userspace/falco/app_actions/load_rules_files.cpp

```diff
@@ -43,59 +45,29 @@ void application::check_for_ignored_events()
 
 	/* Get the events we consider interesting from the application state `ppm_sc` codes. */
 	std::unique_ptr<sinsp> inspector(new sinsp());
-	auto interesting_events = inspector->get_event_set_from_ppm_sc_set(m_state->ppm_sc_of_interest);
-	std::unordered_set<uint32_t> ignored_events;
+	std::unordered_set<uint32_t> events(rule_events.begin(), rule_events.end());
 
-	for(const auto& it : rule_events)
+	auto event_names = inspector->get_events_names(events);
```

Comment:
This should work, instead of declaring another set in the previous line :thinking:
 
```suggestion
	auto event_names = inspector->get_events_names(std::unordered_set<uint32_t>(rule_events.begin(), rule_events.end()));
```